src/pudl/transform/params/ferc1.py

```diff
@@ -1833,20 +1847,72 @@
             "opex_per_kwh": PERKWH_TO_PERMWH,
             "net_generation_kwh": KWH_TO_MWH,
         },
-        "drop_invalid_rows": {
-            "invalid_values": [0, pd.NA, np.nan],
-            "allowed_invalid_cols": [
-                "record_id",
-                "utility_id_ferc1",
-                "plant_name_ferc1",
-                "report_year",
-                "entity_id",
-                "date",
-                "start_date",
-                "end_date",
-                "OrderNumber",
-                "PlantName",
-            ],
-        },
+        "drop_invalid_rows": [
+            {
+                "invalid_values": [0, "0", pd.NA, np.nan, "", "none"],
+                "allowed_invalid_cols": [
+                    "record_id",
+                    "utility_id_ferc1",
+                    "plant_name_ferc1",
+                    "report_year",
+                    "date",
+                    "start_date",
+                    "end_date",
+                    "order_number",
+                    "plant_name",
+                    "plant_type",
+                ],
+            },
+            {
+                "invalid_values": [
+                    pd.NA,
+                    np.nan,
+                    "",
+                ],
+                "required_valid_cols": ["plant_name_ferc1"],
+            },
+        ],
+        # "spot_fix": [
```

Comment:
these are spot fixes for records that have `""` plant names but could be identified relatively easily. I plan on going back in and implementing a spot-fixer.